code/modules/reagents/chemistry/reagents/paradise_pop.dm

```diff
@@ -120,7 +120,7 @@
 /datum/reagent/consumable/drink/grape_granade/on_mob_life(mob/living/M)
 	if(prob(1))
 		var/turf/simulated/T = get_turf(M)
-		goonchem_vortex(T, FALSE, 0, TRUE) // Ignore the 0 volume
+		goonchem_vortex(T, FALSE, 100) //Capped at 100 to prevent sorium abuse
```

Comment:
this still means that you can throw 20 items, 20 hatchets does 300 damage. might I suggest making it scale with the number of units of reagent inside of you?

Reply:
I didn't do reagent scaling because I was concerned that someone who casually drinks it would have most of it metabolize before the burp triggered, making the effect lame/nonexistent. It'd also allow powergamers to chug _more_ than 100u and make the issue worse.

I've lowered the volume to 30, which should have a negligible effect on gameplay even in a worst-case scenario. Thanks for the suggestion!